kuma/static/js/libs/ckeditor/source/plugins/wordcount/README.md

```diff
@@ -0,0 +1,92 @@
+CKEditor-WordCount-Plugin
+=========================
+
+WordCount Plugin for CKEditor v4 (or above) that counts the words/characters an shows the word count and/or char count in the footer of the editor.
+
+![Screenshot]
+(http://www.watchersnet.de/Portals/0/screenshots/dnn/CKEditorWordCountPlugin.png)
+
+####Demo
+
+http://w8tcha.github.com/CKEditor-WordCount-Plugin/
+
+DISCLAIMER: This is a forked Version, i can not find the original Author anymore if anyone knows the original Author please contact me and i can include the Author in the Copyright Notices. 
+
+####License
+
+Licensed under the terms of the MIT License.
+
+####Installation
+
+ If building a new editor using the CKBuilder from http://ckeditor.com/, there is no need to follow numbers steps below. If adding the Word Count & Char Count plugin to an already established CKEditor, follow the numbered steps below.
+
+1.	Download the Word Count &  Char Count plugin from http://ckeditor.com/addon/wordcount or https://github.com/w8tcha/CKEditor-WordCount-Plugin. This will download a folder named **wordcount_*version*.zip** or **CKEditor-WordCount-Plugin-master.zip** to your Downloads folder.
+2.	Download the Notification plugin from http://ckeditor.com/addon/notification. This will download a folder named **notification_*version*.zip** to your Downloads folder.
```

Comment:
Yes, it's only required if you set max lengths, which we won't be doing. I'll add that comment though.